config/locales/es/budgets.yml

```diff
@@ -39,6 +43,9 @@ es:
       balloting: Votación final
       reviewing_ballots: Votación finalizada
       finished: Resultados
+    voting_style:
```

Comment:
Could you change this Spanish translations to:

```
voting_style:
  knapsack: Bolsa de dinero
  approval: Por votos
```

🙏 😌

Reply:
Will do